internal/host/transition_test.go

```diff
@@ -6139,6 +6139,7 @@ var allValidationIDs []validationID = []validationID{
 	CompatibleAgent,
 	NoSkipInstallationDisk,
 	NoSkipMissingDisk,
+	NoIPCollisions,
```

Comment:
Let's add a test that we actually move to insufficient when this validation fails.

See existing `"Moves from known to insufficient when disk skip validations fail - no skip installation disk"` test and its siblings, should be trivial to do the same for your validation